src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerFieldsTest.php

```diff
@@ -392,6 +393,11 @@ public function testCGetActionAllWebspaceNodes()
         $result = json_decode($client->getResponse()->getContent(), true);
 
         $layer = $result['_embedded']['nodes'];
+
+        usort($layer, function ($layer1, $layer2) {
+            return strcmp($layer1['title'], $layer2['title']);
+        });
```

Comment:
The document initializer is being used, this was probably before I
sorted the webspaces in the initializer. In which case removing
this would depned on that PR.

On Thu, Apr 14, 2016 at 02:27:07AM -0700, Daniel Rotter wrote:
>    In
>    [1]src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerFieldsTest.php:
> 
>  > @@ -392,6 +393,11 @@ public function testCGetActionAllWebspaceNodes()
>  >          $result = json_decode($client->getResponse()->getContent(), true);
>  >
>  >          $layer = $result['_embedded']['nodes'];
>  > +
>  > +        usort($layer, function ($layer1, $layer2) {
>  > +            return strcmp($layer1['title'], $layer2['title']);
>  > +        });
> 
>    Why does this have to be sorted now? Was there some behavior changed?
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu/pull/2277#discussion_r59687126
>    2. https://github.com/sulu/sulu/pull/2277/files/542841adaeb64b3060c415c9e532c563daf4c328#r59687126
>    3. https://github.com/sulu/sulu/pull/2277/files/542841adaeb64b3060c415c9e532c563daf4c328#r59687126